static/bidder-info/displayio.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: "https://prebid.display.io/?publisher={{.PublisherID}}"
+endpointCompression: gzip
+geoscope:
+  - global
+maintainer:
+  email: contact@display.io
```

Comment:
Prebid team has sent email to verify above maintainers email. Requesting to responding back on email thread

Reply:
received response from `contact@display.io`